cfme/tests/infrastructure/test_vmware_provider.py

```diff
@@ -249,7 +254,31 @@ def test_vmware_cdrom_dropdown_not_blank():
             4.Virtual machine is selected
             5.Dropdown of ISO files is not empty for CD/DVD Drive
     """
-    pass
+    datastore_collection = appliance.collections.datastores
+    ds = [ds.name for ds in provider.data['datastores'] if ds.type == 'iso']
+    try:
```

Comment:
**Question** Why did you go this route vs have an `if/else` here. I feel it would be better to check if the list is empty vs try to instantiate something that isn't there.

Reply:
I don't have preference here. In either case I'd skip the test on `else` or `expect`. Do you think it will call the function before it checks if `ds[0]` actually exists and valid?

Reply:
I think it be better to check if ds is not None and then try to instantiate the first index

Reply:
leaving this as is, discussed offline